jib-maven-plugin/README.md

```diff
@@ -266,12 +267,12 @@ Property | Type | Default | Description
 `volumes` | list | *None* | Specifies a list of mount points on the container.
 `workingDirectory` | string | *None* | The working directory in the container.
 
-<a name="extradirectory-object"></a>`extraDirectory` is an object with the following properties (see [Adding Arbitrary Files to the Image](#adding-arbitrary-files-to-the-image)):
+<a name="extradirectories-object"></a>`extraDirectories` is an object with the following properties (see [Adding Arbitrary Files to the Image](#adding-arbitrary-files-to-the-image)):
 
-Property | Type
---- | ---
-`path` | string
-`permissions` | list
+Property | Type | Default | Description
+--- | --- | --- | ---
+`paths` | list | `[(project-dir)/src/main/jib]` | List of extra directories. Can be absolute or relative to the project root.
+`permissions` | list | *None* | Permission map for the files added from extra directories.
```

Comment:
We should explicitly say that it maps from file path on container to unix permissions, and maybe even say that it defaults to 644 for files and 755 for folders.